cherab/tools/observers/group/spectroscopic.py

```diff
@@ -0,0 +1,267 @@
+
+# Copyright 2016-2021 Euratom
+# Copyright 2016-2021 United Kingdom Atomic Energy Authority
+# Copyright 2016-2021 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
+
+import matplotlib.pyplot as plt
+from numpy import ndarray
+
+from cherab.tools.observers.group.base import Observer0DGroup
+from raysect.optical import Spectrum
+from raysect.optical.observer import SpectralRadiancePipeline0D, SpectralPowerPipeline0D, RadiancePipeline0D
+
+
+class SpectroscopicObserver0DGroup(Observer0DGroup):
```

Comment:
I kept the class so that we can have SpectroscopicGroup for backward compatibility. Sublcasses were also added to this file so that all the depracated classes would be here.